OpenMEEGConfigure.h.in

```diff
@@ -27,3 +27,4 @@ static const char version[] = "@OpenMEEG_VERSION@";
 #cmakedefine USE_OPENBLAS
 #cmakedefine USE_VECLIB
 #cmakedefine CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION "@CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION@"
+#cmakedefine WIN32
```

Comment:
Locally at least none of the `#if WIN32` conditionals were actually working on Windows. I fixed this by adding this line, and then changing all `#if WIN32` to `#ifdef WIN32`. Let's see if CIs agree...